projects/base/Brewfile

```diff
@@ -11,7 +11,7 @@ cask 'google-chrome'
 cask 'google-drive-file-stream'
 cask 'iterm2'
 cask 'barklyjava8'
-cask 'slack'
+# cask 'slack'
```

Comment:
Rather than commenting out, you can go ahead and delete these lines

Reply:
@steve450  updated